### PR TITLE
Dart quickstart and tutorial: copyedits, markdown and code-excerpt cleanup

### LINKED ISSUE #93: Dart example code: drop unnecessary const and new keywords
cc @mit-mit @sigurdm

### PR DESCRIPTION
**Pull request by chalin**
- Contributes to #87 and #88 for Dart.
- Fixes #93

### CODE REVIEW

**Comment by chalin** [comment]
Possibly best reviewed by ignoring whitespace changes:
> image

**Comment by chalin** [comment]
@mit-mit @sigurdm might you want to review this PR too?

File: content/docs/quickstart/dart.md [hunk 1]
```diff
@@ -26,17 +26,17 @@ binaries for your operating system (`protoc-<version>-<os>.zip`) from here:
...
-Next, install the protoc plugin for Dart
+Next, install the `protoc` plugin for Dart
 
 ```sh
 $ pub global activate protoc_plugin
 ```
 
 The compiler plugin, `protoc-gen-dart`, is installed in `$HOME/.pub-cache/bin`.
-It must be in your $PATH for the protocol compiler, protoc, to find it.
+It must be in your `PATH` for the protocol compiler, protoc, to find it.
```
Comment (sigurdm) [inline content/docs/quickstart/dart.md]: Should this be 
```
`$PATH`
```
to make clear that this is a bash variable?

**Comment by chalin** [comment]
Thanks for the prompt review @sigurdm!

File: content/docs/quickstart/dart.md (hunk 1 above)
Comment (chalin) [reply content/docs/quickstart/dart.md]: In our Dart & Flutter docs, AFAIK, we use the environment variable _name_ only (as opposed to the expression `$foo`). For example, this is excerpt from [Running an app with the standalone Dart VM](https://dart.dev/tutorials/server/cmdline#running-an-app-with-the-standalone-dart-vm):

> Important: The location of the SDK installation directory ... **By putting this directory in your PATH** you can refer to the dart command and other commands, such as dartanalyzer, by name.

@kwalrath can correct me if I'm wrong.

**Comment by mit-mit** [comment]
Thanks much @chalin !

File: content/docs/quickstart/dart.md (hunk 1 above)
Comment (sigurdm) [reply content/docs/quickstart/dart.md]: Good to know!

**Comment by menghanl** [comment]
cc @srini100 to sanity check and approve

**Pull request merged**